dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -1291,21 +1291,33 @@ public boolean canCreateNewVersion(Context context, Item item) throws SQLExcepti
      */
     @Override
     public List<MetadataValue> getMetadata(Item item, String schema, String element, String qualifier, String lang) {
-        List<MetadataValue> dbMetadataValues = super.getMetadata(item, schema, element, qualifier, lang);
+        if (StringUtils.equals(schema, MetadataSchemaEnum.RELATION.getName()) && !StringUtils.equals(element, "type")) {
```

Comment:
I included a comment for now, but it would be good to migrate relationship.type to a constant. I would recommend we create a JIRA ticket for that (similar to the migration of the relationship schema constant)